markhor_tracks/src/markhor_hw_interface.cpp

```diff
@@ -79,9 +99,10 @@ void MarkhorHWInterface::setupCTREDrive()
     front_left_drive->Config_kF(0, 0, timeout_ms_);
     front_left_drive->Config_kP(0, tracks_kp, timeout_ms_);
     front_left_drive->Config_kI(0, tracks_ki, timeout_ms_);
-    front_left_drive->Config_kD(0, 0, timeout_ms_);
-    front_left_drive->ConfigMaxIntegralAccumulator(0, integral_max, timeout_ms_);
-    front_left_drive->Config_IntegralZone(0, integral_zone, timeout_ms_);
+    front_left_drive->Config_kD(0, tracks_kd, timeout_ms_);
+    front_left_drive->ConfigMaxIntegralAccumulator(0, tracks_i_max, timeout_ms_);
+    front_left_drive->Config_IntegralZone(0, tracks_i_zone, timeout_ms_);
+    front_left_drive->ConfigSelectedFeedbackCoefficient(1.0 / 25.0, 0, timeout_ms_); 
```

Comment:
I think you're missing the `tracks_fb_coeff` here and for every other tracks also.

Reply:
Yup nice catch!🙂